[DOCS-7297] Integrate ADW and APS (#1334)
Alfresco/docs-alfresco: digital-workspace/latest/process/starting.md

    Once the process starts you can view the running process by clicking **View running process** at the bottom of the window.

   > **Note:** You can also start a process by right clicking a file in your search results and selecting **Start Process**.

> ## Badges

Reviewer: I would call it `Part of running process badge` since it's about specific badge not about the mechanism in general

> Badges are icons visible next to your documents that are currently in a workflow. To check the processes involved with the document, complete the following steps:

Reviewer: ```suggestion
Badges are icons visible next to your documents, they are designed to represent different statuses of them. Part of running process badge is visible for documents that are currently in a workflow. To check the processes involved with the document, complete the following steps:
```

> Badges are icons visible next to your documents, they are designed to represent different statuses of them. Part of running process badge is visible for documents that are currently in a workflow. To check the processes involved with the document, complete the following steps:

Reviewer 2: ```suggestion
Badges are icons visible next to your documents, and they are designed to represent different statuses of them. The **Part of running process** badge is visible for documents that are currently in a workflow. To check the processes involved with the document, complete the following steps:
```

1. Click on the running processes badge.

    A side panel containing all the metadata opens.

2. Click on **Linked Process** to view a list of all the currently running processes of that particular document. 

    The name of all the linked processes along with their starting date is displayed. 

> 3. Click on **View Details** to view more information about the tasks completed.

Reviewer: ```suggestion
3. Click on **View Details** to view more information about the specific process.
```
